src/Avalonia.Controls/Presenters/ScrollContentPresenter.cs

```diff
@@ -227,6 +230,72 @@ protected override Size ArrangeOverride(Size finalSize)
             return finalSize;
         }
 
+        // Arbitrary chosen value, probably need to ask ILogicalScrollable
+        private const int LogicalScrollItemSize = 50;
```

Comment:
Note: this value was taken from mouse wheel scrolling: https://github.com/AvaloniaUI/Avalonia/blob/551ce9b058afc572f112252dd8ea8bb44cdd8301/src/Avalonia.Controls/Presenters/ScrollContentPresenter.cs#L312